content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -213,8 +213,6 @@ Multus supports all [reference plugins](https://github.com/containernetworking/p
 
 ### NSX-T
 
-[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
```

Comment:
I'd leave in
```markdown
[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform.
```
for context, so that readers can get an insight into whether the plugin is relevant to their cluster design. If we force them to use Google to find out that NSX-T (pretty much) requires VMware, we're not really helping the reader as much as we could.